<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>719</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1291</width>
      <height>681</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>İşlemler</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>1291</width>
        <height>421</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Plaka Okuma Yazdırma</string>
      </property>
      <widget class="QGroupBox" name="gbcikti">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>821</width>
         <height>281</height>
        </rect>
       </property>
       <property name="title">
        <string>İşlem Çıktıları</string>
       </property>
       <widget class="QLabel" name="label_kp">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>70</y>
          <width>321</width>
          <height>201</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;img src=&quot;:/newPrefix/4.jpg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>30</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Crop :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>70</y>
          <width>331</width>
          <height>201</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;img src=&quot;:/newPrefix/4.jpg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>30</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Croplu Görsel :</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="gbtespit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>821</width>
         <height>101</height>
        </rect>
       </property>
       <property name="title">
        <string>Modele Göre Tespit</string>
       </property>
       <widget class="QPushButton" name="plakatespit">
        <property name="geometry">
         <rect>
          <x>620</x>
          <y>40</y>
          <width>181</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Plaka Tespit</string>
        </property>
       </widget>
       <widget class="QComboBox" name="modelselect">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>40</y>
          <width>181</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>VGG16</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CNN</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>30</y>
          <width>121</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Model Seçimi :</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="gbyaz">
       <property name="geometry">
        <rect>
         <x>850</x>
         <y>30</y>
         <width>401</width>
         <height>151</height>
        </rect>
       </property>
       <property name="title">
        <string>Plaka Yazdırma</string>
       </property>
       <widget class="QPushButton" name="printlisance">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>111</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Yazdır</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="plakayazi">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>80</y>
          <width>191</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>50</y>
          <width>51</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Plaka :</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1291</width>
        <height>221</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Görsel Seçimi - Model Eğitimi</string>
      </property>
      <widget class="QGroupBox" name="gbegit">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>20</y>
         <width>461</width>
         <height>191</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Derin Öğrenme Eğitimi</string>
       </property>
       <widget class="QPushButton" name="Btn_modelTrain">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>100</y>
          <width>351</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>CNN Derin Öğrenme - Model Oluştur</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Btn_modelTrain_2">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>40</y>
          <width>351</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>VGG16 Derin Öğrenme - Model Oluştur</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="gbacc">
       <property name="geometry">
        <rect>
         <x>1010</x>
         <y>20</y>
         <width>271</width>
         <height>191</height>
        </rect>
       </property>
       <property name="title">
        <string>ACC - LOSS - SCORE</string>
       </property>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>71</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>ACC/LOSS :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>ACC SCORE :</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="modelacclosstb">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>251</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QTextEdit" name="modelacctb">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>251</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="gbsec">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>511</width>
         <height>191</height>
        </rect>
       </property>
       <property name="title">
        <string>Görsel Seçimi</string>
       </property>
       <widget class="QPushButton" name="Btn_selectFile">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>60</y>
          <width>91</width>
          <height>81</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Görsel Seç</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>20</y>
          <width>261</width>
          <height>141</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;img src=&quot;:/newPrefix/4.jpg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Grafikler</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>441</width>
        <height>561</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string>ACC/LOSS GRAF</string>
      </property>
      <widget class="QLabel" name="accgraf">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>361</width>
         <height>211</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;img src=&quot;:/newPrefix/4.jpg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="lossgraf">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>320</y>
         <width>361</width>
         <height>211</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;img src=&quot;:/newPrefix/4.jpg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ACC GRAF:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>290</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>LOSS GRAF:</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
